api/graphql/schema.graphql

```diff
@@ -532,6 +537,17 @@ type Role {
   displayName: LocalizedString @rename(attribute: "display_name")
   description: LocalizedString
   isTeamBased: Boolean @rename(attribute: "is_team_based")
+  roleAssignments: [RoleAssignment!]
+    @hasMany
+    @can(ability: "viewAnyTeamMembers", model: "Team")
```

Comment:
> I've added the guard to roleAssignments but this one is a bit strange since many roles have roleAssignements but there is no team to query a policy from. I could make a 'viewAnyRoleMembers' policy on Roles and use that instead?

I missed this comment on my last review - yes, I think you nailed it with your suggestion to just make a new policy method on Role 👍 . We'll make a new permission to go along with it and give it to Platform Admins. Call it either view-any-roleMembers or view-any-roleAssignments.

Reply:
Sounds good.  I've added it in [Add role policy to view any assignment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/3ade4ec6d4cddb8d3436c52457874b539ada2f1e).